src/Raven.Server/Documents/PeriodicBackup/PeriodicBackupRunner.cs

```diff
@@ -320,6 +323,68 @@ public long StartBackupTask(long taskId, bool isFullBackup)
             return CreateBackupTask(periodicBackup, isFullBackup, SystemTime.UtcNow);
         }
 
+        public async Task DelayAsync(long taskId, TimeSpan delay, X509Certificate2 clientCert)
+        {
+            foreach (var periodicBackup in _periodicBackups)
+            {
+                var runningTask = periodicBackup.Value.RunningTask;
+                if (runningTask == null || runningTask.Id != taskId)
+                    continue;
+
+                var delayUntil = DateTime.UtcNow.AddTicks(delay.Ticks);
+
+                var command = new DelayBackupCommand(_database.Name, RaftIdGenerator.NewId())
+                {
+                    TaskId = periodicBackup.Key,
+                    DelayUntil = delayUntil
+                };
+
+                try
+                {
+                    (long index, _) = await _database.ServerStore.SendToLeaderAsync(command);
+                    await _database.ServerStore.WaitForCommitIndexChange(RachisConsensus.CommitIndexModification.GreaterOrEqual, index);
+                }
+                catch (Exception e)
+                {
+                    if (_logger.IsOperationsEnabled)
+                    {
+                        var msg =
+                            $"Fail to delay backup task with task id '{taskId}' cluster-wide, the task was delayed until '{delayUntil}' UTC only on the current node.";
+                        
+                        _logger.Operations(msg, e);
+                    }
```

Comment:
shouldn't we rethrow here? We failed to send the delay command and we will proceed anyway?

Reply:
Yes, we can't delay backup cluster-wide for some reason (cluster down, for example), but we still need to kill operation and delay ongoing task on the current node.